AppExamples/CleverDeal.React/src/index.tsx

```diff
@@ -5,10 +5,15 @@ import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 const DEFAULT_ORIGIN: string = "corporate.symphony.com";
+// uncomment if you run against st3 pod
+// const DEFAULT_PARTNER_ID: number = 3;
 const originInParams = (new URL(window.location.href)).searchParams.get('ecpOrigin');
 
 const loadSdk = (
 ): Promise<void> => {
+  // uncomment if you run against st3 pod
+  // (window as any).symphonyPartnerId = DEFAULT_PARTNER_ID; 
```

Comment:
If we have the new partnerId query param, we can add it to the sdk script as `data-partner-id` (or something close to that 😅  ) attribute. 
`sdkScriptNode.setAttribute('data-partner-id', partnerIDFromQueryParam)`